src/store/entities.js

```diff
@@ -107,7 +106,7 @@ function entitiesPending(state, { meta }) {
 function clearFilters(state) {
     return {
         ...state,
-        activeFilters: [defaultFilters]
+        activeFilters: []
```

Comment:
We should be very careful about changing things in the the redux store, since some applications are expecting certain things to be in there and those would need to be reworked before. Same for below.

Reply:
@mkholjuraev Thank you for double checking!